props/prop_base.h

```diff
@@ -631,8 +637,18 @@ class PropBase : CommandParser, Looper, protected SaberBase {
 #ifdef SAVE_PRESET
     ResumePreset();
 #else
-    SetPreset(0, false);
-#endif
+    SetPreset(0, false); // DoNewFont or not? set to announce if yes desired.
+    // Have Blade ID play bladein and bladeout sounds.
+#endif // SAVE_PRESET
+#ifndef BLADE_DETECT_PIN
+    PVLOG_DEBUG << "************** Blade status = " << (bladestatus ? "IN" : "OUT") << "\n";
+    if (system_booted) {
+      SaberBase::DoBladeDetect(bladestatus);
```

Comment:
I think we can simplify this by putting this line in PollScanId(), right after FindBlade